### ISSUE TITLE
[Bug]: No interface instructions for partition.delete on milvus website 

### ISSUE THREAD

**Issue by binbinlv** [milvus-io/milvus#15460]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version:NA
- Deployment mode(standalone or cluster):NA
- SDK version(e.g. pymilvus v2.0.0rc2):NA
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

No interface instructions for partition.delete on milvus website 

### Expected Behavior

Within interface instructions for partition.delete on milvus website 

### Steps To Reproduce

```markdown
https://milvus.io/api-reference/pymilvus/v2.0.0/api/partition.html#pymilvus.Partition.delete

And no instructions for delete() in API References
```

**Comment by yanliang567** [comment]
/assign @XuanYang-cn 
@XuanYang-cn please help to fix this issue.
/unassign

**Comment by XuanYang-cn** [comment]
PR merged, need to cherry-pick to 2.0 branch.

**Comment by XuanYang-cn** [comment]
Cherry-picked, should be updated on the official website shortly

/unassign
/assign @binbinlv
